tensorboard/plugins/graph/tf_graph_common/hierarchy.ts

```diff
@@ -48,10 +48,17 @@ import * as tf_graph from './graph';
 import * as tf_graph_proto from './proto';
 import * as tf_graph_util from './util';
 
+export enum HierarchyEvent {
+  /**
+   * Fired when the templates may have been updated. No event payload attached.
+   */
+  TemplatesUpdated = 'TemplatesUpdated',
```

Comment:
ENUMs are often uppercased. `TEMPLATES_UPDATED = 'TemplatesUpdated',`.

Also, when the value of the enum is not useful, use the default numeric one.

Reply:
Thanks, done.